tests/test_inference_problem.py

```diff
@@ -39,20 +40,42 @@ def test_shared_latent_evaluate(self):
 
 class TestVBProblem(unittest.TestCase):
     def test_prior(self):
-        p = VariationalBayesProblem()
-        p.add_model_error(ModelError())
+        p = InferenceProblem()
+        key = p.add_model_error(ModelError())
         p.define_shared_latent_parameter_by_name("B")
-        p.set_normal_prior("B", 0.0, 1.0)
-        self.assertRaises(Exception, p.set_normal_prior, "not B", 0.0, 1.0)
+        p.set_parameter_prior("B", 0.0, 1.0)
```

Comment:
this is not readable (the user has to know that this is a normal distribution. So either directly using the scipy stats function or kwargs would make it much more transparent.

Reply:
How about two separate methods like
* `p.set_parameter_prior("B", scipy.stats....)`
* `p.set_parameter_normal_prior("B", 0.0, 1.0)`
Technically, it is not needed, but convenient.

Reply:
Would be fine with me, but why not directly use the scipy.stats.normal, then it is also clear what each parameter means. (would also be one method less to maintain :).